components/GovernancePower/Vanilla/useDepositCallback.tsx

```diff
@@ -86,6 +93,11 @@ export const useDepositCallback = (
         sendingMessage: 'Depositing tokens',
         successMessage: 'Tokens have been deposited',
       })
+
+        // Force the UI to recalculate voter weight
+        queryClient.invalidateQueries({
```

Comment:
:-( ideally this would happen naturally due to auto-invalidation, but whatever

Reply:
True - we are still seeing issues here (also in prod) - I will leave it in for now but will review in the milestone 3 PR.